include/unicorn/x86.h

```diff
@@ -8,6 +8,15 @@
 extern "C" {
 #endif
 
+//Memory-Management Register fields (idtr, gdtr, ldtr, tr)
+//borrow from SegmentCache in qemu/target-i386/cpu.h
+typedef struct x86_mmr {
+    uint32_t selector;
+    uint64_t base;   /* handle 32 or 64 bit CPUs */
+    uint32_t limit;
+    uint32_t flags;
+} x86_mmr;
+
```

Comment:
i can see that you are trying to use the same struct for all related instructions. however, IDTR & GDTR do not have `limit` and `flags`, so this is confused. should we have separate structs for them?